### PR TITLE
add a quick check of LVI mitigation tool

### PR DESCRIPTION
**Pull request by jinghe-INTC**
To avoid the user reporting issues for the same reason, such as https://github.com/intel/intel-sgx-ssl/issues/78, https://github.com/intel/intel-sgx-ssl/issues/64, and https://github.com/intel/intel-sgx-ssl/issues/57, even it has been described in the https://github.com/intel/intel-sgx-ssl/blob/master/README.md.

### CODE REVIEW

**Comment by jinghe-INTC** [comment]
If do `make `or  `make all` in an environment without mitigation toolset, it will be like below:
```
$ make
Please follow the instruction at https://github.com/intel/linux-sgx/blob/master/README.md#build-the-intelr-sgx-sdk-and-intelr-sgx-psw-package to install "mitigation tools".
Makefile:41: recipe for target 'sgxssl' failed
make: *** [sgxssl] Error 1
```
and it is OK to workaround the checking by doing `make sgxssl_no_mitigation`, in case there were no proper toolset available.

File: Linux/Makefile [hunk 1]
```diff
@@ -31,13 +31,16 @@
...
+EMPTY_SRC = check_lvi_toolset.cpp
 .PHONY: sgxssl sgxssl_no_mitigation clean install uninstall
 
 
 all: sgxssl
 
 sgxssl:
+	@rm -rf $(EMPTY_SRC) && touch $(EMPTY_SRC)
+	@gcc -Wa,-mlfence-after-load=yes -c $(EMPTY_SRC) 2> /dev/null \
+		|| (echo "Please follow the instruction at https://github.com/intel/linux-sgx/blob/master/README.md#build-the-intelr-sgx-sdk-and-intelr-sgx-psw-package to install \"mitigation tools\" first."  &&  exit 1)
```
Comment (lzha101) [inline Linux/Makefile:43]: Suggest to remove the EMPTY_SRC after check.

Comment (jinghe-INTC) [reply Linux/Makefile:43]: Yes. The possible object file will be also deleted.

Comment (jinghe-INTC) [reply Linux/Makefile:43]: Done.

**Review by lzha101 — APPROVED** [review]

**Pull request merged**